### PR TITLE
Update code for detached container and to match new website tutorial

### PR DESCRIPTION
**Pull request by micahgodbolt**
changed code over to support detached container coming in 0.47, updated some code to better align with tutorial written in FF repo.

### CODE REVIEW

**Comment by micahgodbolt** [comment]
obviously not going to build yet. we can wait to merge until release comes out.

**Comment by tylerbutler** [comment]
I think you meant 0.47?

File: tsconfig.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
         "outDir": "built",
         "sourceMap": true,
         "inlineSources": true,
-        "strict": true,
+        "strict": false,
```
Comment (tylerbutler) [inline tsconfig.json:11]: why's this needed?

Comment (micahgodbolt) [reply tsconfig.json:11]: turning off the typings so it doesn't yell at all of the any's......happy to take suggestions of a better way

File: package.json [hunk 2]
```diff
@@ -13,8 +13,8 @@
         "start:server": "npx tinylicious"
     },
     "dependencies": {
-        "@fluid-experimental/fluid-framework": "^0.45.0",
-        "@fluid-experimental/frs-client": "^0.45.0",
+        "fluid-framework": "^0.48.0",
+        "@fluidframework/azure-client": "^0.48.0",
```
Comment (micahgodbolt) [inline package.json:17]:
```suggestion
        "fluid-framework": "^0.47.0",
        "@fluidframework/azure-client": "^0.47.0",
```

File: src/app-azure.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { SharedMap } from "fluid-framework";
+import { AzureClient, InsecureTokenProvider } from "@fluidframework/azure-client";
+import { vueRenderView as renderDiceRoller } from "./view";
```
Comment (skylerjokiel) [inline src/app-azure.js:8]: Curious why Vue is the default?

**Comment by micahgodbolt** [comment]
Hah, because that was the last one I was testing when I committed 😄 I'll make JS the default.


Sent from Outlook<http://aka.ms/weboutlook>

________________________________

File: src/app-azure.js (hunk 3 above)
Comment (micahgodbolt) [reply src/app-azure.js:8]: oh, forgot to update the azure one. fixed it in the main app.js

File: src/app-azure.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+// const prodConfig: AzureConnectionConfig = {
+//     tenantId: "",
+//     tokenProvider: new AzureFunctionTokenProvider("", azureUser),
+//     orderer: "",
+//     storage: "",
+// };
+
+const client = new AzureClient(localConfig);
+const containerConfig = {
+    initialObjects: { dice: SharedMap }
```
Comment (skylerjokiel) [inline src/app-azure.js:35]: This code doesn't run. You need `diceMap` here or `dice` below

```suggestion
    initialObjects: { diceMap: SharedMap }
```

File: src/view/jsView.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export const jsRenderView = (diceMap, elem) => {
+    const dataKey = 'value';
```
Comment (skylerjokiel) [inline src/view/jsView.js:7]:
```suggestion
    const dataKey = 'dice-value-key';
```

File: src/view/jsView.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    elem.append(wrapperDiv);
+
+    const dice = document.createElement('div');
+    dice.style.fontSize = '200px';
+
+    const rollButton = document.createElement('button');
+    rollButton.style.fontSize = '50px';
+    rollButton.textContent = 'Roll';
+    // Set the value at our dataKey with a random number between 1 and 6.
+    rollButton.onclick = () => diceMap.set("value", Math.floor(Math.random() * 6)+1);
```
Comment (skylerjokiel) [inline src/view/jsView.js:20]:
```suggestion
    rollButton.onclick = () => diceMap.set(dataKey, Math.floor(Math.random() * 6)+1);
```

File: src/view/jsView.js (hunk 5 above)
Comment (skylerjokiel) [reply src/view/jsView.js:7]: Using the word value as the key for a value is pretty confusing.

File: src/view/reactView.jsx [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+ */
+
+import React from "react";
+import ReactDOM from "react-dom";
+
+export const reactRenderView = (dice, elem) => {
+    ReactDOM.render(<ReactView dice={dice} />, elem);
+}
+
+const ReactView = (props) => {
```
Comment (skylerjokiel) [inline src/view/reactView.jsx:13]: can you add a key const here too maybe. Something that is not the word "value"

File: src/view/vueView.js [hunk 8]
```diff
@@ -4,13 +4,12 @@
  */
 
 import { createApp } from 'vue';
-import { IRenderView } from "../types";
 /**
  * Render Dice into a given HTMLElement as a text character, with a button to roll it.
- * @param dataObject - The Data Object to be rendered
- * @param div - The HTMLElement to render into
+ * @param dice - The DDS to be rendered
```
Comment (skylerjokiel) [inline src/view/vueView.js:9]:
```suggestion
 * @param dice - The SharedMap holding the collaborative data
```

File: src/view/vueView.js [hunk 9]
```diff
@@ -21,30 +20,30 @@ export const  vueRenderView: IRenderView = (data, div) => {
                 Roll
             </button>
         </div>`,
-        data: () => ({ diceValue: 1 }),
+        data: () => ({ diceValue: dice.get("value") }),
```
Comment (skylerjokiel) [inline src/view/vueView.js:23]: Same here. Add a const. Also let's be consistent with our quotes though a file.

**Review by skylerjokiel — APPROVED** [review]
Code looks good. Left some cleanup nits.

File: src/view/vueView.js (hunk 9 above)
Comment (micahgodbolt) [reply src/view/vueView.js]: yeah. set up a single key in the app and import that key into each view

**Pull request merged**